### PR TITLE
Documentation package

### PR DESCRIPTION
**Pull request by Nicowcow**
Here's the updated documentation. I'll have to rebase before merging, but @zimbatm it would be nice if you could have a look.

### CODE REVIEW

File: man/cide.1.md [hunk 1]
```diff
@@ -91,6 +101,17 @@ https://docs.docker.com/docker/reference/commandline/cli/#environment-variables:
...
+
+The following environment variables are used by the Amazon `s3` SDK to
+authenticate, and should be set by the user. See also
+http://docs.aws.amazon.com/sdkforruby/api/index.html.
+
+*S3_USER*
+  The Amazon s3 username
+
+*S3 PASS*
+  The amazon s3 pass
```
Comment (zimbatm) [inline man/cide.1.md]: The following environment variables are used by the AWS S3 uploader to authenticate. See also
 +http://docs.aws.amazon.com/sdkforruby/api/index.html.

AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION

we also use the AWS_BUCKET to select which S3 bucket to upload the package to

**Comment by zimbatm** [comment]
@Nicowcow nice job !

File: man/cide.1.md (hunk 1 above)
Comment (Nicowcow) [inline man/cide.1.md]: :+1:

**Comment by zimbatm** [comment]
Cherry-picked into the package branch

**Pull request closed without merging**